### DISCUSSION TITLE
NER transformers example pipeline

### DISCUSSION THREAD

**Discussion by AdrianKrebs** [explosion/spaCy#10652] [Help: Coding & Implementations]
I've used the ner_demo (https://github.com/explosion/projects/tree/v3/pipelines/ner_demo) as the basis for my NER pipeline and it worked well so far.
To further increase the accuracy, I want to switch to fine-tuning a transformer model. I read through the documentation but couldn't find any bootstrapped project configuration (similar to ner_demo). 
Because I'm struggling to migrate the project.yml and the generated config.cfg to `en_core_web_trf`, I was wondering if there are any project.yml samples for transformer models out there?

**Comment by adrianeboyd** [comment]
If you just run `spacy project run all`, you can add `-G` to the `create-config` command to generate a config with `transformer`+`ner`. You'll want to set the `gpu_id` at the top before training for reasonable training speeds (although I think this toy example will still train relatively quickly on CPU if you just want to try it once; we wouldn't recommend training on CPU for non-toy dataset sizes).

```yaml
    script:
      - "python -m spacy init config --lang ${vars.lang} --pipeline ner configs/config.cfg -G --force"
```

(The optional `train-with-vectors` command that's not part of `all` will fail because the generated config doesn't have a `tok2vec` in it to enable vectors for.)
